client/ayon_core/tools/loader/control.py

```diff
@@ -425,3 +432,28 @@ def _create_event_system(self):
 
     def _emit_event(self, topic, data=None):
         self._event_system.emit(topic, data or {}, "controller")
+
+    def get_current_context_product_types_filter(self):
+        context = get_current_context()
```

Comment:
```suggestion
        context = self.get_current_context()
```

Reply:
Is `self.get_current_context()` the context the user browsed to in the UI or the context the UI was launched with? :)

Reply:
Current context of host used in controller.